packages/contracts/contracts/test/MajorityVotingMock.sol

```diff
@@ -5,38 +5,26 @@ pragma solidity 0.8.10;
 import "../voting/majority/MajorityVotingBase.sol";
 
 contract MajorityVotingMock is MajorityVotingBase {
-    function initializeMock(
-        IDAO _dao,
-        uint64 _supportThreshold,
-        uint64 _minParticipation,
-        uint64 _minDuration,
-        uint256 _minProposerVotingPower
-    ) public initializer {
-        __MajorityVotingBase_init(
-            _dao,
-            _supportThreshold,
-            _minParticipation,
-            _minDuration,
-            _minProposerVotingPower
-        );
+    function initializeMock(IDAO _dao, PluginSettings calldata _pluginSettings) public initializer {
+        __MajorityVotingBase_init(_dao, _pluginSettings);
     }
 
     function createProposal(
         bytes calldata, /* _proposalMetadata */
         IDAO.Action[] calldata, /* _actions */
         uint64, /* _startDate */
         uint64, /* _endDate */
-        bool, /* _executeIfDecided */
-        VoteOption /* _choice */
+        bool, /* _tryEarlyExecution */
+        VoteOption /* _voteOption */
```

Comment:
Shouldn't createProposal receive a struct? Or it's just like that because of it being a simple mock?

Reply:
My point is about reducing the stack size, as reported by some external collaborators. Reducing the amount of local variables (params included) helps greatly

Reply:
For now, there is enough room left and in `MajorityVoting` and `Admin` and `Multisig` only need two parameters. Maybe we change this in the future (and release a new plugin version) but for now, it is not urgent.